client/components/domains/use-my-domain/style.scss

```diff
@@ -9,6 +9,26 @@
 	flex-direction: column;
 	padding: 36px 24px;
 	background-color: inherit;
+
+	& &--redesign {
+		padding: 0;
+		background: transparent;
```

Comment:
Can we try to avoid css override? I see that this background comes with that `.card` class from `@automattic/components` package and I'm wonder if we could use the `@wordpress/components` one instead.